test/com/google/javascript/jscomp/CollapsePropertiesTest.java

```diff
@@ -55,7 +56,12 @@ public CollapsePropertiesTest() {
 
   @Override
   protected CompilerPass getProcessor(final Compiler compiler) {
-    return new CollapseProperties(compiler, propertyCollapseLevel);
+    return new CollapseProperties(
+        compiler,
+        propertyCollapseLevel,
+        compiler.getOptions().chunkOutputType,
```

Comment:
If possible, I think it simplifies the unit tests to make these constructor parameters unit test fields rather than part of the CompilerTestCase API. Can we e.g. just make `chunkOutputType` a field  in this file that we only set to `ES_MODULES` in the test case that needs it?

Reply:
Yes. It's a bit weird since its needed both in this test case and in CompilerTestCase, but since we are hard coding it in CompilerTestCase that solves the problem.